### DISCUSSION TITLE
Data contents in claims data

### DISCUSSION THREAD

**Discussion by m-bellaiche** [synthetichealth/synthea#1159] [Q&A]
Hi!  

I wanted to ask two questions regarding the contents of the Claim and ExplanationOfBenefit resources available in the 1k patient sample R4 dump from the Synthea [downloads page](https://synthea.mitre.org/downloads).   

**Q1:  Are there coded diagnosis and procedure information available?**
I can't seem to find coded representations of the diagnoses or procedures associated with these resources, which I would expect to live in the `diagnosis`.`diagnosis` and `procedure`.`procedure` codeableConcept fields for both resources.

For Claim:
- I can see `Claim.diagnosis.sequence` and `Claim.procedure.sequence` populated for some resources (~10% and ~35% respectively), but no information beyond that.
- `Claim.item.xSequence` is populated for both x=diagnosis and procedure, making reference to those array entries.

For ExplanationOfBenefit:
- I see `ExplanationOfBenefit.diagnosis.sequence` or `ExplanationOfBenefit.diagnosis.type` populated for some (~15%) resources, but no ICD codes or any of the procedure fields.
- `ExplanationOfBenefit.item.diagnosisSequence` is populated

**Q2:  Are there explanation of benefits (EOB) available for corresponding pharmacy claims?**
There are ~61k Claim resources vs ~47k ExplanationOfBenefits resources. As far as I can tell, none of the pharmacy claims have an associated EOB, all institutional claims have an associated EOB.  Conversely, all EOB's refer to medical care.  

Thank you!

**Comment by jawalonoski** [comment]
The download you linked to is from September 2019. We probably should update the Downloads because things have changed since then, although not much (if any) for FHIR Claims/EOB in particular. 

# FHIR Claim

> Q1: Are there coded diagnosis and procedure information available?

Yes.

The way our code works:

- `Claim.diagnosis` is only populated if there is a **new** `Condition` diagnosed during the `Encounter`.
- `Claim.procedure` is only populated if there is a `Procedure` performed within the `Encounter`.
- The "coded" information is either in the resources referenced by `Claim.diagnosis` or `Claim.procedure` (within the same patient Bundle), or is captured in the `Claim.item[x].productOrService.coding`.
- If none of these are coded, the claim is for the `Encounter` code... no _billable_ procedures were performed other than the visit itself (which is billable), and no new diagnosis was made.
- Sometimes an `Encounter` can have a `reasonCode` which could be a new diagnosis or an old diagnosis.
- _All medications are in separate claims_. For medications/pharmacy claims, find `Claim.type.coding[x].code == 'pharmacy'`. The medication prescribed is linked by reference within `Claim.prescription.reference` to another resource within the `Bundle`,  where you can get the RxNorm code.
- If you want to find the diagnosed Condition related to the prescription, you have to follow that `Claim.prescription.reference` link and then follow the `MedicationRequest.reasonReference`.

# FHIR ExplanationOfBenefit

> Q2: Are there explanation of benefits (EOB) available for corresponding pharmacy claims?

No. An `ExplanationOfBenefit` is only generated for "encounter claims" and not "medication claims." I'm not sure the reasoning behind that -- it could be an oversight.

> "... but no ICD codes or any of the procedure fields."

Synthea does not use ICD codes. Synthea mostly uses RxNorm, LOINC, and SNOMED.
